app/controllers/planning_applications/assessment/policy_areas/parts_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Assessment
+    module PolicyAreas
+      class PartsController < BaseController
+        before_action :ensure_can_assess_planning_application
+        before_action :find_policy_parts
+
+        def index
+          respond_to do |format|
+            format.html
+          end
+        end
+
+        private
+
+        def ensure_can_assess_planning_application
+          render plain: "forbidden", status: :forbidden and return unless @planning_application.can_assess?
+        end
+
```

Comment:
I think this can be removed as it's duplicating the base controller now

```suggestion
```

Reply:
good spot!